java/ql/src/Security/CWE/CWE-020/OverlyLargeRange.ql

```diff
@@ -22,6 +23,7 @@ TreeView::RegExpCharacterClass potentialMisparsedCharClass() {
 
 from TreeView::RegExpCharacterRange range, string reason
 where
+  AlertFiltering::filterByLocation(range.getLocation()) and
```

Comment:
As noted in the corresponding issue, I think we should remove this and all similar filtering that cannot be expected to improve performance.

Reply:
This is done.